tests/sample/ext/drv/sample_ext.vcxproj

```diff
@@ -130,11 +130,17 @@
     <ClInclude Include="..\inc\sample_ext_helpers.h" />
   </ItemGroup>
   <ItemGroup>
+    <ProjectReference Include="$(SolutionDir)libs\execution_context\kernel\execution_context_kernel.vcxproj">
+      <Project>{26e7ed0b-c128-4d7c-a90e-c246def40ad3}</Project>
+    </ProjectReference>
     <ProjectReference Include="$(SolutionDir)libs\platform\kernel\platform_kernel.vcxproj">
       <Project>{fc3f9998-4085-4767-8386-5453f07c3aad}</Project>
     </ProjectReference>
+    <ProjectReference Include="$(SolutionDir)libs\ubpf\kernel\ubpf_kernel.vcxproj">
```

Comment:
Why is this even needed?

Reply:
Removed.  It was in line 114 before, but it seems to be fine without it.